plutus-core/executables/Common.hs

```diff
@@ -59,41 +59,49 @@ type PlcProg =
 type UplcProg =
   UPLC.Program PLC.Name PLC.DefaultUni PLC.DefaultFun
 
-class Executable a where
+class Executable p where
```

Comment:
I changed `a` to `p` here because we often use `a` for the annotation type and that was confusing me.